website/pages/en/cookbook/enums.mdx

```diff
@@ -0,0 +1,245 @@
+---
+title: Categorize NFT Marketplaces Using Enums
+---
+
+Learn how to effectively organize information from transactions and marketplace interactions using Enums.
+
+> Note: This guide uses the CryptoCoven NFT smart contract.
+
+## What are Enums?
+
+Enums, or enumeration types, are a specific data type that allows you to define a set of allowed values.
+
+### Enums Syntax
+
+You can define enums in your schema, and once defined, you can use the string representation of the enum value to set an enum field on an entity.
+
+Here's what an enum definition might look like in your schema:
+
+```graphql
+enum TokenStatus {
+  OriginalOwner
+  SecondOwner
+  ThirdOwner
+}
+```
+
+This means that when you use the type `TokenStatus` in your schema, you expect it to be exactly one of `OriginalOwner`, `SecondOwner`, or `ThirdOwner`.
+
+To learn more about enums, check out [Creating a Subgraph](/developing/creating-a-subgraph/#enums) and [GraphQL documentation](https://graphql.org/learn/schema/#enumeration-types).
+
+## Benefits of Using Enums
+
+- **Clarity:** Enums provide meaningful names for values, making data easier to understand.
+- **Validation:** Enums enforce strict value definitions, preventing invalid data entries.
+- **Maintainability:** When you need to change or add new categories, enums allow you to do this in a focused manner.
```

Comment:
How about adding a before & after comparison to highlight these benefits?

eg:
Before using enums, your code could look like this ...
After using enums, it's like this so typos won't affect you and the code is easier to maintain.